shared/src/simulation/activities/index.ts

```diff
@@ -0,0 +1,19 @@
+import { unloadVehicleActivity } from './unload-vehicle';
+
+export const simulationActivities = {
+    unloadVehicleActivity,
+};
+
+export type ExerciseSimulationActivity = typeof unloadVehicleActivity;
+
+export type ExerciseSimulationActivityState = InstanceType<
+    ExerciseSimulationActivity['activityState']
+>;
+
+// TODO: typing
+export const simulationActivityDirectory = Object.fromEntries(
```

Comment:
I would rather name this `[...]Dictionary` to be consistent to the `ExerciseActionTypeDictionary` of the actions and action reducers.